new-frontiers-14/frontier-station-14
# Stelloxadone guidebook fix: do not collapse non-even damage distributions

## About the PR



Fixes descriptions for medical reagents with non-even distributions.  Constant of 0.02 allows for a difference of +/- 0.01, for non-even distributions (e.g. 4 brute turns into 1.33, 1.33, 1.34).

## Why / Balance


People didn't know Stelloxadone cures rads.

## How to test


1. Open guidebook, go to Medical Doctor->Cryogenics
2. Check Stelloxadone.  Wow, explicit damage types!
3. Check Necrosol.  Wow, ~~burn~~ brute damage collapsed into its damage group!

## Media


Fixed version of stelloxadone description.
![stellaaaa](https://github.com/user-attachments/assets/101e4dde-2598-4cce-ac1f-0869a73d0849)

Necrosol with the changes added, brute damage matches original value, collapsed into one amount.
![necroposting](https://github.com/user-attachments/assets/2ee947fc-1fac-4578-8165-430e847f24a2)

## Requirements



## Breaking changes


**Changelog**


:cl:
- fix: The Stelloxadone guidebook description is now accurate to rad and poison damage.
